slack-archive/src/main/java/juja/microservices/slack/archive/controller/ArchiveController.java

```diff
@@ -22,6 +26,9 @@
 @Slf4j
 public class ArchiveController {
 
+    @Inject
+    private ArchiveService service;
```

Comment:
Please use constructor injection it will be better

Reply:
OK, in the next pr I going to fix it.